framework/src/outputs/DOFMapOutput.C

```diff
@@ -52,17 +52,6 @@ DOFMapOutput::DOFMapOutput(const InputParameters & parameters) :
     _system_name(getParam<std::string>("system_name")),
     _mesh(_problem_ptr->mesh())
 {
-  // Set output coloring
-  if (Moose::_color_console)
-  {
-    char * term_env = getenv("TERM");
-    if (term_env)
-    {
-      std::string term(term_env);
-      if (term != "xterm-256color" && term != "xterm")
```

Comment:
Let's not drop this logic. It's been awhile since I've been on a dumb terminal but we really don't want colors on in that case. It'll hose you big time.